tests/mobly/controllers/android_device_lib/adb_test.py

```diff
@@ -712,6 +717,53 @@ def mock_handler(raw_line):
             self.assertEqual(stderr,
                              mock_execute_and_process_stdout.return_value)
 
+    def test_root_success(self):
+        with mock.patch.object(adb.AdbProxy, '_exec_cmd') as mock_exec_cmd:
+            mock_exec_cmd.return_value = MOCK_ROOT_SUCCESS_OUTPUT
+            output = adb.AdbProxy().root()
+            mock_exec_cmd.assert_called_once_with(
+                ['adb', 'root'],
+                shell=False,
+                timeout=None,
+                stderr=None)
+            self.assertEqual(output, MOCK_ROOT_SUCCESS_OUTPUT)
+
+    @mock.patch('time.sleep', return_value=mock.MagicMock())
+    def test_root_success_with_retry(self, mock_sleep):
+        with mock.patch.object(adb.AdbProxy, '_exec_cmd') as mock_exec_cmd:
```

Comment:
I think you can use `@mock` for this instead?
let's make the mock style consistent

Reply:
same for the other tests